### ISSUE TITLE
Remove references of users setting their slurm user account

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal#418]
With the new fractal register command (i.e. https://github.com/fractal-analytics-platform/fractal/issues/395), slurm account association is done by the admin. Is it correct that the user can't control this anymore, i.e. a user can't overwrite that slurm submissions are made in the name of another user (see https://github.com/fractal-analytics-platform/fractal-server/issues/129)?

If yes, we should remove the references to users setting their slurm user from here https://fractal-analytics-platform.github.io/fractal/install.html and here https://fractal-analytics-platform.github.io/fractal/cli.html
And we should clean up the fractal help message:

fractal --help gives
```
(fractal-client-1.0.1) bash-4.4$ fractal --help
usage: fractal [-h] [-u USER] [-p PASSWORD] [-s SLURM_USER] [--verbose] [--batch] [-j]
               {project,dataset,task,workflow,job,version,user} ...

Command-line interface for Fractal Client

optional arguments:
  -h, --help            show this help message and exit
  -u USER, --user USER  User email address for login (overrides configuration file)
  -p PASSWORD, --password PASSWORD
                        User password (overrides configuration file)
  -s SLURM_USER, --slurm-user SLURM_USER
                        SLURM user
  --verbose, -v         Change minimal logging level from default to DEBUG
  --batch               Return output suitable for scripting, e.g., only the ID of items created instead of the full
                        object.
  -j, --json            Output raw json

Commands::
  {project,dataset,task,workflow,job,version,user}
```

But the user shouldn't be setting the SLURM_USER anymore.

If it's still possible for a user to overwrite this via the client, we should remove this ability.

**Comment by tcompa** [comment]
> Is it correct that the user can't control this anymore, i.e. a user can't overwrite that slurm submissions are made in the name of another user (see https://github.com/fractal-analytics-platform/fractal-server/issues/129)?

It is correct, indeed.

> If yes, we should remove the references to users setting their slurm user from here https://fractal-analytics-platform.github.io/fractal/install.html and here https://fractal-analytics-platform.github.io/fractal/cli.html

Agreed.

To do:
- [ ] Fix "install" page in docs 
- [ ] Fix arguments of fractal cli command (cli docs are fixed automatically)

> If it's still possible for a user to overwrite this via the client, we should remove this ability.

It is not, unless you are a superuser. In that case, the fractal user register and fractal user edit commands already have the right arguments/docs:
* https://fractal-analytics-platform.github.io/fractal/cli_files/user.html#register
* https://fractal-analytics-platform.github.io/fractal/cli_files/user.html#edit

**Comment by tcompa** [comment]
Now released with 1.0.2.
